sunpy/net/tests/test_fido.py

```diff
@@ -97,6 +98,14 @@ def test_save_path():
             assert target_dir in f
             assert "eve{}0".format(os.path.sep) in f
 
+    with tempfile.TemporaryDirectory() as target_dir:
```

Comment:
I do wonder if we should split it into two tests.

Reply:
We are downloading the same files. So I figured downloading the files into a different directory would be the best option to test for flaws.

Reply:
Well in that case, do we need to redo the Fido.search part? Could we not just fetch it twice one after the other?

Reply:
Should I split it into two tests? @nabobalis

Reply:
Could we not just fetch the files twice with two different paths?

Reply:
Now the searching is only done once.

Reply:
Can you add a comment somewhere to just state what the difference in the two parts of the test are.

Reply:
It is in a different test now! ^_^